functions/common.ts

```diff
@@ -9,9 +9,11 @@ import type { Issues } from "../.prisma/client";
 
 export interface Assignee {
   name: string;
+  [key: string]: any;
```

Comment:
What is the purpose of this?
```suggestion
  [key: string]: any;
```

Reply:
it allows for additional properties to be present in Assignee objects
there was an error with merging with the main:

TSError: ⨯ Unable to compile TypeScript:
conversion.ts:17:9 - error TS2322: Type 'Assignee[]' is not assignable to type 'JsonValue'.
  Type 'Assignee[]' is not assignable to type 'JsonArray'.
    Type 'Assignee' is not assignable to type 'JsonValue'.
      Type 'Assignee' is not assignable to type 'JsonObject'.
        Index signature for type 'string' is missing in type 'Assignee'.

17         assignees: card.memberNames.map((name) => ({ name } as Assignee)),
           ~~~~~~~~~

Reply:
Do the following:
1. Add the following to .env (in the project's top level folder) `postgresql://postgres:postgres@YOUR_IP:5432/local?schema=public` with `YOUR_IP` replaced with your IP, it should look like 192.168.X.X.
2. These are also needed in the .env for our code to work:
REDIRECT_URI: 'http://localhost:5173/callback'
SERVER_URL: 'https://local.functions.nhost.run/v1'
4. Change directory to 'functions'
5. Run `npm run generate`
6. Run `npm run migrate`

The commands are for generating a database client which includes interfaces for IUnifiedIssues.